FilterViewController.m

```diff
@@ -22,20 +24,56 @@ @implementation FilterViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     
-    MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initalizeManager:UITableViewRowAnimationMiddle title:@"Labels" tableView:self.tableView];
+    MHCollapsibleViewManager *labels = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Labels" tableView:self.tableView];
+    
+    //sends double array for filternames and single array for header lines
+    [labels setDataWithFilterNames:[[NSArray alloc] initWithObjects:[self returnLabelArray],nil] headerTitles:[[NSArray alloc] initWithObjects:@"Labels", nil]];
+    
+    MHCollapsibleViewManager *surveys = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Surveys" tableView:self.tableView];
+    
+    NSArray *surveyAnswers = [[NSArray alloc] initWithObjects:[self returnSurveyArray], [self returnSurveyArray], [self returnSurveyArray], nil];
+    NSArray *surveyQuestions = [[NSArray alloc] initWithObjects:@"Survey 1", @"Survey 2", @"Survey 3", nil];
     //Index 0 is title
-    [labels initializeData:[NSMutableArray arrayWithObjects:@"Freshman", @"Sophomore", @"Junior", @"Senior", nil]];
+     [surveys setDataWithFilterNames:surveyAnswers headerTitles:surveyQuestions];
+    surveyAnswers = nil;
+    surveyQuestions = nil;
     
-    MHCollapsibleViewManager *assignTo = [[MHCollapsibleViewManager alloc] initalizeManager:UITableViewRowAnimationMiddle title:@"Assigned To" tableView:self.tableView];
+    MHCollapsibleViewManager *interactions = [[MHCollapsibleViewManager alloc] initManagerWithAnimation:UITableViewRowAnimationMiddle topHierarchyTitle:@"Interactions" tableView:self.tableView];
     //Index 0 is title
-    [assignTo  initializeData:[NSMutableArray arrayWithObjects:@"Jan", @"Andy", @"Peggy", @"Maggie", nil]];
+    [interactions setDataWithFilterNames:[[NSArray alloc] initWithObjects:[self returnInteractionsArray],nil] headerTitles:[[NSArray alloc] initWithObjects:@"Interactions", nil]];
```

Comment:
This also a very well written class. This is a minor thing. You can write this more concisely with an array literal (literals are the things with @ in front of them. They expand to what you wrote.):
```
[interactions setDataWithFilterNames:@[[self returnInteractionsArray]] headerTitles:@[@"Interactions"]];
```
You could simlify it further using the dot notation on the returnInteractionsArray and a shorter name for the method
```
[interactions setFilterNames:@[self.returnInteractionsArray] headerTitles:@[@"Interactions"]];
```


Note: On the topic of literals, if you see a string with an @ in front it will also expand to [[NSString alloc] initWith...] which is helpful to know so you know what methods are available on those objects.